test/distributed/_sharded_tensor/ops/test_embedding.py

```diff
@@ -51,20 +76,52 @@ def _run_sharded_embedding(self, spec, input_size, num_embeddings, embedding_dim
         inp = torch.randint(0, num_embeddings, tuple(input_size)).cuda(self.rank)
         sharded_output = sharded_embedding(inp)
 
+        # If max_norm is set, we need to ensure that the renorm has been applied across
+        # inputs from all ranks.
+        if max_norm is not None:
+            gathered_inputs = [torch.zeros_like(inp) for _ in range(TEST_GPU_NUM)]
+            dist.all_gather(gathered_inputs, inp)
+            unique_inp = torch.unique(torch.cat(gathered_inputs))
+            local_embedding(unique_inp)
```

Comment:
So here, I apply the inputs from all ranks to the local embedding. Otherwise the test would fail. cc: @pritamdamania87